charts/datadog/values.yaml

```diff
@@ -283,6 +283,10 @@ datadog:
     ## ref: TODO - add doc link
     enabled: false
 
+  networkConfig:
```

Comment:
should we move also `enableConntrack` and `collect_dns_stats` into this new section?

Reply:
That would be a breaking change, and these options should be changed very infrequently. I'd rather leave these here than try to add them to networkConfig in a backwards compatible way.

Reply:
Sorry I haven't used the proper word.
do you think it could be a good idea to deprecate the `systemProbe.enableConntrack` and `systemProbe.collectDNSStats` and to replace them by `networkConfig.enableConntrack` and `networkConfig.collectDNSStats`

Reply:
I don't feel like it's worth the effort of maintaining, these features are both fairly stable and on by default.

Reply:
what do you think about a section like this
```yaml
  networkMonitoring:
    enabled: true
    config:
      // any config parameter here
```